core/src/main/scala/scalafix/util/OrganizeImports.scala

```diff
@@ -7,14 +7,27 @@ import scala.meta._
 import scala.meta.semantic.v1.Completed
 import scala.meta.tokens.Token.Comment
 import scalafix.config.FilterMatcher
-import scalafix.rewrite.ScalafixCtx
 import scalafix.rewrite.RewriteCtx
 import scalafix.syntax._
 import scalafix.util.TreePatch.AddGlobalImport
 import scalafix.util.TreePatch.RemoveGlobalImport
 
-private[this] class OrganizeImports private (implicit ctx: ScalafixCtx) {
-  import ctx._
+/** Set of operations needed to run organize imports */
+trait OrganizeImportsMirror {
+
+  /** Returns true if this importee is never used and can be removed. */
+  def isUnused(importee: Importee): Boolean
+
+  /** Returns fully qualified name of this reference
+    *
+    * For example scala.collection.immutable.List for List.
+    **/
+  def fullyQualifiedName(ref: Ref): Option[Ref]
```

Comment:
I remember there was a problem with fqn not working correctly if implemented via scala.meta's mirror. What's its current status? Is there something I can do on my side?